packages/components/switch/src/switch.vue

```
@@ -184,6 +184,7 @@ const actualValue = computed(() => {
 const checked = computed(() => actualValue.value === props.activeValue)
 
 if (![props.activeValue, props.inactiveValue].includes(actualValue.value)) {
+  debugWarn(COMPONENT_NAME, 'model-value must be active-value or inactive-value')
```

ESLint

[error] Replace `COMPONENT_NAME,·'model-value·must·be·active-value·or·inactive-value'` with `⏎····COMPONENT_NAME,⏎····'model-value·must·be·active-value·or·inactive-value'⏎··`

(prettier/prettier)